File: 1.Know Basic Maths/5.GCD.js
function GCD(n1,n2){

    for(let i = 1;i<=Math.min(n1,n2);i++){
        if(n1%i === 0 && n2%i === 0){
            gcd = i;
        }
    }
    return gcd;
}

function main(){
    let n1 = 15;
    let n2 = 20;

    let gcd = GCD(n1,n2);

    console.log(gcd);
}

main();

File: 1.Know Basic Maths/5.GCD.cpp
#include<bits/stdc++.h>
using namespace std;

int GCD(int n1, int n2){

    int gcd = 1;

    for(int i = 1;i<=min(n1,n2);i++){
        if(n1%i == 0 && n2%i == 0){
            gcd = i;
        }
    }
    return gcd;

} 

// int main(){

    // int n1 = 15;
    // int n2 = 20;

    // int ans = GCD(n1,n2);

    // cout<<ans<<endl;


// }